modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/carbon15051/email_login_config.toml

```diff
@@ -0,0 +1,33 @@
+[server]
+pattern = "DEV"  #possible values, DEV,HA,HA_WITH_ANALYTICS
+hostname = "localhost"
+node_ip = "127.0.0.1"
+base_path = "https://localhost:9853"
+offset = "410"
```

Comment:
Not sure if this a realistic offset number.

Reply:
This is the offset value used in previous tests when running with xml files.

Reply:
OK, then it's fine